ports/microsoft-signalr/CONTROL

```
@@ -0,0 +1,9 @@
+Source: microsoft-signalr
+Version: 0.1.0-alpha1
+Description: C++ Client for ASP.NET Core SignalR.
+Default-Features: cpprestsdk
+Homepage: https://github.com/aspnet/SignalR-Client-Cpp
+
+Feature: cpprestsdk
+Description: Add default Http and WebSocket implementations using CppRestSDK
+Build-Depends: cpprestsdk[default-features,websockets] (!uwp)
```

This should not be `(!uwp)`; we shouldn't silently failing to depend on cpprestsdk in uwp.

```suggestion
Build-Depends: cpprestsdk[default-features,websockets]

```

---

cpprestsdk+websockets doesn't work on uwp, but we want the default to be to install cpprestsdk. So it seems better to silently not install cpprestsdk on uwp than to force everyone else to have to specify cpprestsdk as a feature. (This is because there isn't support for `Supports` on features)

---

Just read your other comment with the fatal_error. That would be ok, especially if I include the correct command to install the package on uwp.

---

Ah, this is how you should do this then:

```
Source: microsoft-signalr
Version: 0.1.0-alpha1
Description: C++ Client for ASP.NET Core SignalR.
Default-Features: default-features
Homepage: https://github.com/aspnet/SignalR-Client-Cpp

Feature: default-features
Build-Depends: microsoft-signalr[cpprestsdk] (!uwp)
Description: Features installed by default

Feature: cpprestsdk
Description: Add default Http and WebSocket implementations using CppRestSDK
Build-Depends: cpprestsdk[default-features,websockets]
```